lib/ssl/src/ssl_internal.hrl

```diff
@@ -74,6 +74,10 @@
 -define(TRUE, 0).
 -define(FALSE, 1).
 
+
+-define(NO_COMPRESSION, ?NULL).
+-define(COMPRESSIONS, [?NO_COMPRESSION]).
```

Comment:
I don't know whether COMPRESSIONS macro is worth having. it is used in only 3 places.
IMHO, when reading code [?NO_COMPRESSION] is more informative, albeit a bit harder to write.